src/redux/sagas/identity.js

```diff
@@ -29,6 +30,15 @@ function* getIdentityWorker(action) {
   }
 }
 
+function* getIdentityOfWorker(action) {
+  try {
+    const identity = yield call(getIdentity, action.payload);
```

Comment:
i believe after the AsyncCopyIconWithAddress is not making dispatches that this can be deleted?